entity-service/src/test/kotlin/com/egm/stellio/entity/authorization/Neo4jAuthorizationServiceTest.kt

```diff
@@ -241,6 +246,100 @@ class Neo4jAuthorizationServiceTest {
         confirmVerified()
     }
 
+    @Test
+    fun `it should return authorized JsonLdEntities without specificAccessPolicy`() {
+        every { neo4jAuthorizationRepository.getSubjectGroups(mockUserUri) } returns setOf(groupUri)
+        every { neo4jAuthorizationRepository.getSubjectRoles(mockUserUri) } returns emptySet()
+        every {
+            neo4jAuthorizationRepository.getAuthorizedEntitiesWithAuthentication(
+                any(),
+                any(),
+                any(),
+                any()
+            )
+        } returns Pair(
+            1,
+            listOf(
+                EntityAccessControl(
+                    id = "urn:ngsi-ld:Beekeeper:1230".toUri(),
+                    type = listOf("Beekeeper"),
+                    createdAt = Instant.now().atZone(ZoneOffset.UTC),
+                    right = AccessRight.R_CAN_READ
+                )
+            )
+        )
+
+        val countAndAuthorizedEntities = neo4jAuthorizationService.getAuthorizedEntities(
+            queryParams = QueryParams(),
+            sub = mockUserSub,
+            offset = offset,
+            limit = limit,
+            contextLink = JsonLdUtils.NGSILD_CORE_CONTEXT,
+            includeSysAttrs = false
+        )
+
+        assertEquals(1, countAndAuthorizedEntities.first)
+        assertEquals(1, countAndAuthorizedEntities.second.size)
+
+        assertTrue {
+            countAndAuthorizedEntities.second.all {
+                it.id.equals("urn:ngsi-ld:Beekeeper:1230".toUri())
```

Comment:
you still have a problem with your conditions (written like this, only the second line will be used in the predicate). You have others in your tests, check them carefully.